src/frontend/src/pages/ChangeRequestDetailPage/ChangeRequestActionMenu.tsx

```diff
@@ -97,6 +96,7 @@ const ChangeRequestActionMenu: React.FC<ChangeRequestActionMenuProps> = ({
         options={users.filter((user) => isLeadership(user.role)).map(taskUserToAutocompleteOption)}
         getOptionLabel={(option) => option.label}
         onChange={(_, values) => setReviewerIds(values.map((value) => value.id))}
+        defaultValue={changeRequest.requestedReviewers.map(taskUserToAutocompleteOption)}
```

Comment:
this kinda works, but you're inserting this in the middle of the data flow in the form, so it's not a great solution

Reply:
Instead, I would pass the `changeRequest.requestedReviewers` in as the defaultValue for the useState for reviewerIds. I would also make the reviewerIds useState a reviewers useState that stores an array of userAutocomplete options, rather than just their ids. From there you should set the value prop on the autocomplete to be the reviewers useState value. You'll need to clean up some other small things to get it to work, but I think this'll be a cleaner solution that works well